app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -595,13 +616,13 @@ class CampaignStacExport(
             _
           )
           ) =>
-        Some((name, ingestLocation))
+        Some(ingestLocation)
```

Comment:
This whole match can now just be `maybeScene flatMap { _.ingestlocation }` -- I can't make a suggestion over all the lines in question but the big messy `case Some(Scene(_, _, _, _, _, ...))` can go away 🎉